test/e2e-upgrade-tests.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Docs -> file://./upgrade/README.md
+
+# Script entry point.
+
+source "$(dirname "$0")/e2e-common.sh"
+
+# Overrides
+
+function knative_setup {
```

Comment:
we did a lot of effort to remove istio and serving from eventing. I don't want to bring it back.

Reply:
Is this being used to hit the endpoint for downloading results?  I have a PR doing somewhat similar things where the least ugly alternative I found (and chose to use) was kubectl port forwarding the test endpoint.  My PR is using the endpoing to allowing full events to be pulled back from target pods for checking event contents against expectations, etc (ie, to switch the CheckLogs calls that grep against printouts in logs to a more structured format).  My PR is here: https://github.com/knative/eventing/pull/2663

Reply:
Istio here is for serving. Let's discuss that in https://github.com/knative/eventing/pull/2388#discussion_r385882618

Reply:
Istio and Serving is now removed